enterprise/server/util/container/converter/converter.go

```diff
@@ -1,4 +1,4 @@
-package container
+package converter
```

Comment:
`converter` as a package name feels slightly generic, wdyt about something like `ociconv` (similar to stdlib's `strconv`) or `imageconv`? (also nest directly under `util` since we usually have a flat package structure for the util dirs)

Reply:
good call, done